pkg/sqlutil/test_data_source.go

```diff
@@ -0,0 +1,17 @@
+package sqlutil
+
+import (
+	"testing"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/proullon/ramsql/driver"
+	"github.com/stretchr/testify/require"
+)
+
+// NewInMemoryDataSource returns a new in-memory DataSource
+func NewInMemoryDataSource(t *testing.T) *sqlx.DB {
```

Comment:
Returning `DataSource` would allow us slip in a hook later without affecting the callers:
```suggestion
func NewInMemoryDataSource(t *testing.T) DataSource {
```